recipe/build-pyarrow.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+set -e
+set -x
+
+# Build dependencies
+export ARROW_HOME=$PREFIX
+export PARQUET_HOME=$PREFIX
+export SETUPTOOLS_SCM_PRETEND_VERSION=$PKG_VERSION
+export PYARROW_BUILD_TYPE=release
+export PYARROW_WITH_DATASET=1
+export PYARROW_WITH_FLIGHT=1
+if [[ "$(uname -m)" = "ppc64le" || "$(uname -m)" = "aarch64" ]]
+then
+  export PYARROW_WITH_GANDIVA=0
+else
+  export PYARROW_WITH_GANDIVA=1
+fi
```

Comment:
We should build this unconditionally with 0.17
```suggestion
export PYARROW_WITH_GANDIVA=1
```

Reply:
Fixed.